client/cypress/integration/dashboard/dashboard_list.js

```diff
@@ -0,0 +1,24 @@
+describe("Dashboard list sort", () => {
+  beforeEach(() => {
+    cy.login();
+  });
+
+  it("creates one dashboard", () => {
+    cy.visit("/dashboards");
+    cy.getByTestId("CreateButton").click();
+    cy.getByTestId("CreateDashboardMenuItem").click();
+    cy.getByTestId("CreateDashboardDialog").within(() => {
+      cy.get("input").type("A Foo Bar");
+      cy.getByTestId("DashboardSaveButton").click();
+    });
+  });
+
+  describe("Sorting table does not crash page ", () => {
```

Comment:
🤔, I don't like much the idea of having a Cypress test for a very specific error case, it would be much better if you could assert that sorting will work as expected, e.g.: `it('can sort table results')`, as this will also cover the "non-error" case

Reply:
btw, I think Arik did Query List Sort tests in the past, I can try to find it 👀

Reply:
This would be good to see. I searched for it but no joy :/